### PR TITLE
[cs] add common set

### PR DESCRIPTION
**Pull request by TomasVotruba**
While I was at upgrading frienly package https://github.com/sabbelasichon/typo3-rector/pull/1404, I've noticed ecs is missing few of basic sets here.

Found a bit time to fix that :)

### CODE REVIEW

File: Classes/Api/CrawlerApi.php [hunk 1]
```diff
@@ -150,24 +126,6 @@ public function addPageToQueue($uid): void
...
-            foreach ($configurations as $confKey => $confArray) {
-                if (!in_array($confKey, $this->allowedConfigurations)) {
-                    unset($configurations[$confKey]);
-                }
-            }
-        }
-
-        return $configurations;
-    }
-
```
Comment (TomasVotruba) [inline Classes/Api/CrawlerApi.php:170]: Methods are now sorted by `public`, `protected`, `private`, that's why many of them are moved.

**Comment by tomasnorre** [comment]
Thanks a lot, have quite a number of branches open atm, so would need to be a little careful with merging. 

But will check, thanks for taking your time, not only here, but also in typo3-rector by Sebastian.

File: Tests/Acceptance/Support/BackendTester.php [hunk 2]
```diff
@@ -39,7 +39,6 @@ class BackendTester extends \Codeception\Actor
 {
     use BackendTesterActions;
     use FrameSteps;
```
Comment (tomasnorre) [inline Tests/Acceptance/Support/BackendTester.php:41]: Which rule triggers this? 
I would prefer an empty line between use in class context, and not have it followed directly by a comment.

**Comment by tomasnorre** [comment]
Would you consider making the request only for the: Sets, Composer and .editorconfig? 
Then I'll easily rebase and apply.

File: Tests/Acceptance/Support/BackendTester.php (hunk 2 above)
Comment (TomasVotruba) [reply Tests/Acceptance/Support/BackendTester.php:41]: It was caused by Slevomat bug, that doesn't take account follow up of comment only. It was just in this file. I've ignored the rule and restored this space.

All the other trait + content are separated by single space correctly.

image

**Comment by TomasVotruba** [comment]
> have quite a number of branches open atm

That's what I love about coding standards, that this doesn't matter. 

After merge, all you need to do is update `ecs.yaml` only and run:

```bash
.Build/bin/ecs check --fix
```

on particular branch before merge. This prevents rebase conflicts and updated code to new style :+1:

File: Tests/Acceptance/Support/BackendTester.php (hunk 2 above)
Comment (tomasnorre) [reply Tests/Acceptance/Support/BackendTester.php]: Ja, that I saw, so was considering why it was like this in this case. 

Thanks.

**Comment by TomasVotruba** [comment]
Woohoo, that was fast :)

:+1: Enjoy

**Comment by TomasVotruba** [comment]
In case of any troubles, let me know

**Comment by tomasnorre** [comment]
Thanks, I like not having contributors waiting. Otherwise they will lose motivation at some point.

**Comment by TomasVotruba** [comment]
Exactly :+1: 

I can do 10 PRs a day, but only 1 in 2 months.

**Pull request merged**